.spellcheckwordlist.txt

```diff
@@ -4,6 +4,7 @@ AdditionalConfiguration
 Adminer
 Apache
 Acquia
+Amazee
```

Comment:
```suggestion
amazee.io
```

The proper wordmark is `amazee.io`. Is this just for the docs? If we removed all references there, then I supposed this could be removed instead.